coursework/mobile/index.md

```diff
@@ -1292,8 +1300,8 @@ Here are some links to get you started on using Lighthouse:
 
 <div class="box">
   <strong class="milestone-counter">Milestone 15:</strong> Achieve a score of at
-  least 12/14 for the Progressive Web App category and include the Lighthouse html report in your
-  repository.
+  least 8/9 for the Progressive Web App category (automated checks only) and include
```

Comment:
The number of checks changed again, from 14 to 9.

Is 8/9 too much to ask of them? The new criteria are:

### Installable
- [ ] Web app manifest and service worker meet the installability requirements

### PWA Optimized
- [ ] Registers a service worker that controls page and `start_url`
- [ ] Redirects HTTP traffic to HTTPS
- [ ] Configured for a custom splash screen
- [ ] Sets a theme color for the address bar.
- [ ] Content is sized correctly for the viewport
- [ ] Has a `<meta name="viewport">` tag with `width` or `initial-scale`
- [ ] Provides a valid `apple-touch-icon`
- [ ] Manifest doesn't have a maskable icon

I think all of them are highly achievable except for the last one.

Reply:
Shall we try 7/9 then? 

Should the last point be "Manifest has a maskable icon" (as the current one is the check failed message?). 

This is new to me :-), but reading around and found that it is not too crazy (sufficient material / tool to allow the teams to achieve this, I think). So should be fine.

Reply:
Ah yes, the current message is the failed one as I was testing with Starbucks' website, and theirs somehow failed it.

Hmm I think 8/9 might be ok for now then? Agreed that most of the items are easily achievable - 7/9 might be a bit too lenient haha